axe_selenium_python/axe.py

```diff
@@ -93,17 +93,19 @@ def report(self, violations):
             string += "\n\n\n"
         return string
 
-    def write_results(self, data, name="results.json"):
```

Comment:
There should be a default parameter so that `write_results` can be called without a filename or path.